setup.js

```diff
@@ -1,2 +1,36 @@
 import { matchers } from "jest-json-schema";
 expect.extend(matchers);
+expect.extend({
+  // If anything goes wrong with a test that requires an account that was auto generated
+  // We write the account used in the message along side all the other accounts that were
+  // generated during the current run
+  // This extends the signers support test previous truthy and falsy expectation matchers
+  signersAssertion(token, logs, accountsUsed, accountPool) {
```

Comment:
So this will print the keypairs used for this test, but whats the purpose of printing the entire account pool? Your changes already add that functionality to `beforeAll()`.

Reply:
The key reason is redundancy and convenience.
Whenever we generate accounts we want to output their keypairs to stdout.
So when you go to circles
But when a failure occurs the logs and messages are saved to json and displayed on [circle's steps screen](https://app.circleci.com/pipelines/github/stellar/transfer-server-validation-runner/11802/workflows/3a63cfc8-6ae7-4780-8119-2fba975d3451/jobs/130231/steps) you'll see the keys in the "Run Tests" stdout dropdown

However if there is an error then we want to output that error, the accounts relevant for that specific test and for verbosity's sake all of the generated accounts in [circle's "tests" screen](https://app.circleci.com/pipelines/github/stellar/transfer-server-validation-runner/11802/workflows/3a63cfc8-6ae7-4780-8119-2fba975d3451/jobs/130231/tests) 

If for whatever reason you need more than just those specific accounts used in that test (or you wish to take the error output and parse the all keypairs for recovery or cleanup)  you do not need to go back and forth between the two screens (steps and tests or download both outputs)

Reply:
I'm not sure I understand. Everything outputted in our tests is in the `Run Tests` step, including both the `Generated keypair ...` statements and any failed test outputs. Why would you need to go back and forth from two UI pages?

Reply:
Addressed 0c1ba36.
The new helper will not output all the generated keypairs.
Your advice is sound, if we wanted the entirety of the verbose output (generated keypairs and errors) we'll go to the steps screen and output the steps stdout to disk.